app/templates/layouts/_questionnaire.html

```diff
@@ -39,8 +37,10 @@
         <button class="btn btn--secondary js-btn-save" data-qa="btn-save-sign-out" type="submit" name="action[save_sign_out]" {{helpers.track('click', 'Navigation', 'Save and complete later click')}}>{% block save_sign_out_button_text %}{{ _("Save and complete later") }}{% endblock %}</button>
       </div>
     </form>
-
 {% endblock main %}
 {% block footer %}
+<div class="container u-mt-xl u-pt-l">
+{% include theme('partials/feedback/expandable_inline.html') %}
+</div>
```

Comment:
I think this has now been moved outside of the form? so now when we submit we get a HTTP 401 as its missing the csrf token

Reply:
The form data wasn't getting sent so I've had to make a fix in the pattern library -> https://github.com/ONSdigital/sdc-global-design-patterns/pull/170

(Feel free to review the code)